src/nft/types/sell/sell.ts

```diff
@@ -59,8 +59,14 @@ export interface Listing {
 
 export interface WalletAsset {
   id?: string
-  image_url: string
-  image_preview_url: string
+  imageUrl: string
+  smallImageUrl: string
+  address: string
```

Comment:
address is unnecessary with asset_contract.address

Reply:
ah you're right thanks for catching